Feat: Add information about two factor authentication and ssh (#91)
virtualidentityag/dev-fe-onboarding: src/fundamentals-1.md

- Git Flow: https://codeburst.io/trunk-based-development-vs-git-flow-a0212a6cae64

### Challenge
#### Create an account on GitHub
If you do not have an account at GitHub.com already, go to: https://github.com/join and create an account.
Please note that GitHub is a professional social network and choose an appropriate username.

> And since you will soon be added to the <i>virtualidenityag</i> organization on GitHub, it's required that you enable two-factor authentication as a extra layer of protection. To do it, got to *settings > security*. There are three methods you can choose from (Authenticator app, Security keys and SMS). Just pick the one you're more comfortable with.

Reviewer: ```suggestion
And since you will soon be added to the <i>virtualidenityag</i> organization on GitHub, it's required that you enable two-factor authentication as a extra layer of protection. To do so, go to *settings > security*. There are three methods you can choose from (Authenticator app, Security keys and SMS). Just pick the one you're most comfortable with.
```
